app/src/main/java/com/example/hangouts/homeScreen/UserUI.java

```diff
@@ -0,0 +1,25 @@
+package com.example.hangouts.homeScreen;
+
+import com.example.hangouts.models.User;
+import com.parse.ParseUser;
+
```

Comment:
Let's put a [javadoc](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html) comment here as to what this class is - something like:
/**
 * UserUiModel - a model for getting user UI data.
 */